code/datums/materials/basemats.dm

```diff
@@ -270,6 +270,18 @@ Unless you know what you're doing, only use the first three numbers. They're in
 	qdel(source.GetComponent(/datum/component/hot_ice, "plasma", amount*150, amount*20+300))
 	return ..()
 
+/datum/material/metalhydrogen
+	name = "Metal Hydrogen"
+	id = "metal hydrogen"
+	desc = "Solid metallic hydrogen. Some say it should be impossible"
+	color = "#f2d5d7"
+	alpha = 150
+	categories = list(MAT_CATEGORY_RIGID = TRUE)
+	sheet_type = /obj/item/stack/sheet/mineral/metal_hydrogen
+	value_per_unit = 0.75
+	beauty_modifier = 0.75
+	strength_modifier = 1.2
+	armor_modifiers = list("melee" = 1.5, "bullet" = 1.5, "laser" = 1.5, "energy" = 1.5, "bomb" = 1.5, "bio" = 1.5, "rad" = 1.5, "fire" = 1.5, "acid" = 1.5)
```

Comment:
too much imo, this is on par with admin materials.

Reply:
well it's a rare enough material that you can't make in high quantities and have just few uses for now, i calculated that you could make a stack of this in more than 1 hour and half into the game. So i dont see the problem with those values being high.

Reply:
you dont need a stack to make armor

Reply:
it shits on material balance by making it so the best material available (in literaly everything) is an atmos thing you can just keep producing

Reply:
just to show how this doesn't work well; you'd need 6 minutes to make OP armor

Reply:
6 minutes? to make this metal? how

Reply:
just a fyi
- make bz farm 
- make electolyzer (wait for science and mining to do their thing)
- burn plasma with low o2 to make h2o and use electrolizer to make h2
- burn plasma with a lot of o2 to make trit
- combine h2, trit and plasma and start fusion
- use bz and a lot of h2 in a spaced chamber and heat up the gases to over 1e6 K and 1e6 kpa
- consume a lot of h2 to make the metal in one go

Reply:
> you could make a stack of this in more than 1 hour and half into the game. So i dont see the problem with those values being high.


5 sheets required
50 / 10 = 5
60 / 10 = 6

6 minutes.

But if what you're saying is it just takes long to set up to have permanent income of the metal, thats just as bad, now evryone has OP armor a while into the round.

Reply:
name one good reason why your material should be better than an admin-only material

Reply:
because if you manage to make it you deserve it? is an end game goal for atmos that requires the knowledge to make fusion, bz farm, hydrogen farm, trit farm and h2o farm. There must be an incentive to make this high end, high tech piece of metal otherwise what's the reason to make it. Also why is bad that you need a long setup? And you dont have a permanent income because you still need to redo everything before you can make more

Reply:
5 sheets required
50 / 10 = 5
60 / 10 = 6

6 minutes.
also what's this?? how did you come up with those numbers

Reply:
you said it takes 60 minutes to make a stack, you only need 5 sheets.

Reply:
I don't think material balance needs an overkill material that is made through atmos that gives you admin-material level stats in literaly /every/ category.

Reply:
if maintaners ask me to nerf this very rare, very hard to get material, i'll do it. also 60 minutes to get one stack in one go (and probably is even less than one stack)

Reply:
nerfed, now is like titanium, but wil more fire resistance (1.3) and less bomb resistance (0.7)